File: pysrc/serve.py
#!/usr/bin/env python3

import os, time, sys, subprocess

if len(sys.argv) < 2:
    print("Usage: python serve.py <chunk-file>")
    sys.exit(1)

chunk_file = sys.argv[1]

while True:
    subprocess.run(["../luasrc/send.lua", chunk_file])
    time.sleep(1)

File: luasrc/serve.lua
#!/usr/bin/env lua

local socket = require("socket")

if #arg < 1 then
    print("Usage: lua serve.lua <chunk-file>")
    os.exit(1)
end

local chunk_file = arg[1]

while true do
  os.execute("../gosrc/send " .. chunk_file)
  socket.sleep(1)
end
